### PR TITLE
Update Calico to v3.27.0

### PR DESCRIPTION
**Pull request by hakman**

### CODE REVIEW

File: tests/e2e/kubetest2-kops/deployer/up.go [hunk 1]
```diff
@@ -292,9 +292,11 @@ func (d *deployer) IsUp() (bool, error) {
 		d.KopsBinaryPath, "validate", "cluster",
 		"--name", d.ClusterName,
 		"--count", strconv.Itoa(d.ValidationCount),
-		"--interval", d.ValidationInterval.String(),
 		"--wait", wait.String(),
 	}
+	if d.ValidationInterval > 10*time.Second {
+		args = append(args, "--interval", d.ValidationInterval.String())
+	}
```
Comment (hakman) [inline tests/e2e/kubetest2-kops/deployer/up.go:299]: For compatibility with older kOps versions that don't support this new flag

Comment (hakman) [reply tests/e2e/kubetest2-kops/deployer/up.go:299]: @upodroid ^^^

Comment (upodroid) [reply tests/e2e/kubetest2-kops/deployer/up.go:299]: Noticed it today :(

**Review by upodroid — APPROVED** [review]
LGTM

**Pull request merged**